### PR TITLE
[Contacts]Screen will flash once or twice after you tap "Find duplica…

### PR DESCRIPTION
**Pull request by arcturus**
…te contacts" r=borjasalguero

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=bc11473bcc56367c7b5df044e281623a9cb6b3d7

File: apps/communications/contacts/views/matching/js/matching_controller.js [hunk 1]
```diff
@@ -63,11 +63,14 @@
...
-        Matcher.match(contact, 'active', callbacks);
-      }, abort);
+      // Give some time to see the searching for duplicates message
+      setTimeout(function() {
+        parentWindow.ContactsService.get(cid, function success(mContact) {
+          // Master contact
+          contact = mContact;
+          Matcher.match(contact, 'active', callbacks);
+        }, abort);
+      }, 1000);
```
Comment (borjasalguero) [inline apps/communications/contacts/views/matching/js/matching_controller.js]: Is this timing working properly in a low end device?

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6514474c050f5d87d0c561795ddb4228c21b8375

**Pull request merged**